scripts/spyglass

```diff
@@ -103,7 +103,9 @@ run_spyglass() {
     --autofocus "${AUTO_FOCUS:-continuous}" \
     --lensposition "${FOCAL_DIST:-0.0}" \
     --autofocusspeed "${AF_SPEED:-normal}" \
-    --orientation_exif "${ORIENTATION_EXIF:-h}"
+    --orientation_exif "${ORIENTATION_EXIF:-h}"\
+    --tuning_filter "${TUNING_FILTER:-}"\
+    --tuning_filter_dir "${TUNING_FILTER_DIR:-}"
```

Comment:
I tested the service now and it seems like it needs the `None` value as default here. So it would be `--tuning_filter "${TUNING_FILTER:-None}"` and `--tuning_filter_dir "${TUNING_FILTER_DIR:-None}"`.
I don't know what bash makes without it specified here, but the service fails with only the tuning file specified, atm.

I think it might pass just an empty string. I don't know how to debug a service file, so I cannot look into it more deeply right now.

Reply:
oh, you have a good point, I will test it out tomorrow, have not tested with only 1 argument. Thanks for checking. -- also add a test case to check 👍

Reply:
Unfortunately test case is kinda useless because we are not testing how we execute it via sh, however I added anyways :D 

Fixed params passing only necessary parameters using dic.

Reply:
This test case shouldn't be bad